bin/BuildPackages.sh

```diff
@@ -190,6 +191,14 @@ build_one_package() {
   (  # start subshell
   set -e
   cd "$CURDIR/$PKG"
+  local PKG_NAME=$($GAPROOT/gap -q -T <<GAPInput
+Read("PackageInfo.g");
+Print(GAPInfo.PackageInfoCurrent.PackageName);
+GAPInput
```

Comment:
Note that this adds an overhead which for people with a HDD can be multiple seconds. Since this is called for every package right now, I am worried that it might add to several minutes of delay when building all packages (granted, only in the worst case; on a machine with SSD, the overhead is usually subseconds.
Two things to improve this come to mind:
1. pass the `-A` to minimize the number of packages being loaded (each adds some extra delay)
2. Only execute this command if we really need it. I.e., move it to inside `run_configure_and_make`, just before the line which actually accesses `CONFIGFLAGS`.